Cargo.toml

```diff
@@ -41,6 +41,8 @@ macros = ["ctor", "indoc", "inventory", "paste", "pyo3cls", "unindent"]
 # Optimizes PyObject to Vec conversion and so on.
 nightly = []
 
+no-return-notimplemented = []
```

Comment:
I think unfortunately this cannot be a feature, because it can change behaviour of other user's code.

Imagine one crate A which uses this feature depends on another crate B which does not. Then suddenly all crate B's APIs change behaviour when compiled as a dependency of crate A.

Reply:
Actually, it doesn't do much right now but opting-out the tests, the actual code would still return NotImplemented.  I found that trying to have the switch led to very ugly code in my hands ;)

I'll remove it.